docs/projects/hello-world/_posts/2019-10-20-arkscript.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Hello World in Arkscript
+layout: default
+last-modified: 2018-10-22
+featured-image: hello-world-in-python-featured-image.JPEG
+tags: [arkscript, hello-world]
+authors:
+  - arjitg
+---
+
+Welcome to this originally first edition of the Hello World in Every Language
+series where I plan to embark on a journey of coding language exploration. 
+First up, how to implement Hello World in Python. t
+-Its really easy. Let's just get into it
```

Comment:
Should be ArkScript, not Python, and there is a trailling t (and a `-` which I think shouldn't be there :) )